include/circt/Dialect/SMT/SMTIntOps.td

```diff
@@ -119,4 +119,13 @@ def IntCmpOp : SMTIntOp<"cmp", [Pure]> {
   let assemblyFormat = "$pred $lhs `,` $rhs attr-dict";
 }
 
+def Int2BVOp : SMTOp<"int2bv", [Pure]> {
+  let summary = "Convert an integer to an n-wide bitvector";
```

Comment:
Can you add a small description describing what happens if the integer does not fix into the BV of the given width?
Are negative integers converted to two-complement?
Since all these conversions are not very standardized, it'd be nice to describe the corner cases in a bit more detail. Since you lower it 1-1 to Z3, I guess that'd just be a copy-paste of Z3's spec?

Reply:
Will do!